### PR TITLE
Migrated to maven

### PR DESCRIPTION
**Pull request by msentwickler**
## Related Issue
#56 

## Type of change

- [x] Bug fix
- [x] Feature request
- [ ] Documenting code

## Checklists

- [x] I referenced the according Issue
  - [x] I created an Issue
  - [x] I put a link in the Section [Related Issue](#related-issue)
- [ ] I tested my code with multiple browsers
- [x] My commit messages are good and understandable

### CODE REVIEW

File: checkstyle.sh [hunk 1]
```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env sh
```
Comment (steve-hb) [inline checkstyle.sh:1]: do not delete this

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  </repositories>
+  <pluginRepositories>
+    <pluginRepository>
+      <id>spring-milestones</id>
+      <name>Spring Milestones</name>
+      <url>https://repo.spring.io/milestone</url>
+    </pluginRepository>
+  </pluginRepositories>
+
+</project>
```
Comment (steve-hb) [inline pom.xml]: Indentation wrong: we are using tabs!

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>de.the-morpheus.e-du</groupId>
+  <artifactId>gateway-service</artifactId> <!-- D NOT CHANGE this! -->
+  <version>0.1-SNAPSHOT</version> <!-- D NOT CHANGE this! -->
+  <name>gateway-service</name> <!-- D NOT CHANGE this! -->
+  <description>Gateway-Microservice</description>
+
+  <properties>
+    <java.version>1.8</java.version>
```
Comment (steve-hb) [inline pom.xml]: we do not have to set this since we are using a compiler plugin

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.0.M4</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>de.the-morpheus.e-du</groupId>
+  <artifactId>gateway-service</artifactId> <!-- D NOT CHANGE this! -->
+  <version>0.1-SNAPSHOT</version> <!-- D NOT CHANGE this! -->
+  <name>gateway-service</name> <!-- D NOT CHANGE this! -->
```
Comment (steve-hb) [inline pom.xml]: Why is the name lowercase but the description is the name uppercase?

Comment (steve-hb) [inline pom.xml]: These comments can be removed

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+  <description>Gateway-Microservice</description>
+
+  <properties>
+    <java.version>1.8</java.version>
+  </properties>
+
+  <dependencies>
```
Comment (steve-hb) [inline pom.xml]: please sort these dependencies and rearrange them in blocks

File: pom.xml (hunk 3 above)
Comment (steve-hb) [reply pom.xml]: ok, we don't? Does this even compile without springs own plugin?

Comment (steve-hb) [reply pom.xml]: otherwise we have to specifiy - somehow - the source and target level

**Review by steve-hb — CHANGES_REQUESTED** [review]

File: checkstyle.sh (hunk 1 above)
Comment (msentwickler) [reply checkstyle.sh:1]: I deleted the whole file.

File: pom.xml (hunk 3 above)
Comment (msentwickler) [reply pom.xml]: I added the maven compiler plugin

**Review by steve-hb — APPROVED** [review]

**Pull request merged**